tests/test_examples.py

```diff
@@ -30,9 +30,12 @@ def _paths_to_strs(x: Iterable[pathlib.Path]) -> Sequence[str]:
 EXAMPLES_DIR = THIS_DIR / ".." / "examples"
 TUTORIALS_DIR = THIS_DIR / ".." / "docs" / "tutorials"
 
-SH_PATHS = _paths_to_strs(EXAMPLES_DIR.glob("*.sh"))
+EXCLUDED_EXAMPLE_FILES = ["train_dagger_atari_interactive_policy.py"]
```

Comment:
Probably one could think about an alternative where we mock parts of the example script etc. However, it does not seem to be super useful, since we have unit tests that check analogous things that this mocked version would check.